zerotier/plan.sh

```diff
@@ -0,0 +1,43 @@
+pkg_name=zerotier-one
```

Comment:
Does it make sense to rename the plan or the directory, so they are in alignment?    (i.e both zerotier or zerotier-one)

Reply:
Oh dang, good catch, probably the directory. I wrestled with what to call it and went back and forth a couple times, I settled on `zerotier-one` because that seems to be what they call the client application. They don't really present it at all in the outward marketing but it's the package name for the client builds they distribute and the name of the command it installs.